src/traits.rs

```diff
@@ -249,6 +250,9 @@ where
 }
 
 /// A distributed algorithm that defines a message flow.
+///
+/// Many algorithms require an RNG which must be supplied on each call. It is up to the caller to
+/// ensure that these source supply random numbers suitable for cryptographic applications.
```

Comment:
"this source supplies" or "these sources supply"?

Reply:
I believe "to ensure that this RNG is cryptographically secure".